### PR TITLE
[Wallet] Add redux migration v0

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

In prep for the wallet release today, we need a migration to make sure our existing users don't have to redo the onboarding flow

### Tested

Created a new test to confirm migration. 
Tested by running dev app with old version and then upgrading to latest.

### Backwards compatibility

Compatible

### CODE REVIEW

**Review by sallyjyl — APPROVED** [review]

File: packages/mobile/src/redux/store.ts [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
 import { rootSaga } from 'src/redux/sagas'
 
 const persistConfig: any = {
   key: 'root',
-  version: -1, // default is -1, increment as we make migrations
+  version: 0, // default is -1, increment as we make migrations
   storage,
   blacklist: ['home', 'geth', 'exchange', 'networkInfo', 'alert'],
   stateReconciler: autoMergeLevel2,
+  migrate: createMigrate(migrations, { debug: true }),
```
Comment (jeanregisser) [inline packages/mobile/src/redux/store.ts:17]: Shouldn't `debug` be false?

**Review by jeanregisser — COMMENTED** [review]
Looks good! great to see the tests you added 👍

Just a clarification, are `app.inviteCodeEntered` and `app.numberVerified` still used?
i.e. I'm wondering if those 2 should be migrated to `undefined`

File: packages/mobile/src/redux/store.ts (hunk 1 above)
Comment (jmrossy) [reply packages/mobile/src/redux/store.ts:17]: Debug just adds some logging, lets leave it as true for now:
https://github.com/rt2zz/redux-persist/blob/208c6b112ead87b3701dfacaae2cdbe78377775a/src/createMigrate.js

**Comment by jmrossy** [comment]
> Looks good! great to see the tests you added 👍
> 
> Just a clarification, are `app.inviteCodeEntered` and `app.numberVerified` still used?
> i.e. I'm wondering if those 2 should be migrated to `undefined`

We could do that yep. Does that get us any benefit?

**Comment by jeanregisser** [comment]
I don't know yet, I'm mostly confused because I still see them in the app reducer. And your changes and comment suggests these are obsolete.

https://github.com/celo-org/celo-monorepo/blob/0319b93d72b15c7486951f6e75bc62532ba4cfe4/packages/mobile/src/app/reducers.ts#L5

What's the reality?

**Comment by jmrossy** [comment]
@jeanregisser They are obsolete but they will not be removed until we clean up the old onboarding flow, which we should do in the next week or two

**Review by jeanregisser — APPROVED** [review]

**Pull request merged**